sormas-backend/src/main/java/de/symeda/sormas/backend/user/UserService.java

```diff
@@ -82,34 +85,50 @@ public User getByUserName(String userName) {
 		Root<User> from = cq.from(getElementClass());
 		cq.where(cb.equal(from.get(User.USER_NAME), userNameParam));
 
-		TypedQuery<User> q = em.createQuery(cq)
-				.setParameter(userNameParam, userName);
+		TypedQuery<User> q = em.createQuery(cq).setParameter(userNameParam, userName);
 
-		User entity = q.getResultList().stream()
-				.findFirst()
-				.orElse(null);
+		User entity = q.getResultList().stream().findFirst().orElse(null);
 		return entity;
 	}
 
 	public List<User> getAllByRegionAndUserRoles(Region region, UserRole... userRoles) {
+		return getAllByRegionAndUserRoles(region, Arrays.asList(userRoles), null);
+	}
+
+	public List<User> getAllByRegionAndUserRolesInJurisdiction(Region region, UserRole... userRoles) {
+		return getAllByRegionAndUserRoles(region, Arrays.asList(userRoles), this::createJurisdictionFilter);
+	}
+
+	private List<User> getAllByRegionAndUserRoles(
```

Comment:
Again, same problem as in the app; this could potentially lead to unexpected and undesired behaviour. We probably need actual pseudonymisation instead.